### DISCUSSION TITLE
Critical bug: 500 error on production: INTERNAL_MISSING_RESPONSE_FROM_CACHE

### DISCUSSION THREAD

**Discussion by amfriedman** [vercel/community#732] [Help] [answered]
I've visited my production website through a custom domain, hosted on Vercel. It has been working fine for months. But last week and again just now the site is *DOWN* with a 500 error and INTERNAL_MISSING_RESPONSE_FROM_CACHE. When I visit the internal Vercel link to the production deployment, the site is fine. But through the custom/vanity domain (yes it is properly registered on the Vercel platform) I get this blocking 500 error. The help docs merely tell me to contact Vercel support. OK, I've sent two support requests to Vercel tech support (starting Thursday) but I have not received a response. I'm on a pro plan which promises 24-48 hours. Working hours? Normal hours? The policy is vague. This is scary! What is the solution? How do I prevent this going forward? (For client confidentiality purposes I cannot reveal the domain or internal link on this forum.)

**Comment by swarnava** [comment]
Hi, Can you raise a support request using https://vercel.com/support/request with deployment url?

**Reply by amfriedman** [reply]
You should delete your answer as I had already stated in my post that I raised two support requests. You did not leave an answer that should be accepted as you do not actually answer any of my questions.

**Reply by swarnava** [reply]
Let us review this quickly and get back to you

**Comment by Pieparker** [comment, marked as answer]
Hey @amfriedman! Firstly, we appreciate your feedback regarding response times. Although it is our Enterprise customers who benefit from guaranteed responses under SLAs, we do strive to respond to Pro customers as punctually and effectively as possible.

Moreover, we've acknowledged your cases in our help desk and, given the sensitivity of your client data/domains, we will continue to follow up with you in case 00090212!

**Comment by amfriedman** [comment]
Well, the dreaded 500 error INTERNAL_MISSING_RESPONSE_FROM_CACHE just appeared again. Third time in two weeks. This is really bad.

**Comment by lmachens** [comment]
I am facing the same issue lately and opened a case (00122869).
It happens on https://www.arkesia.gg/.

**Reply by amyegan** [reply]
Hi Leon. I looked back on the original case for clues. The problem was caused by large request size when fetching from `getStaticProps()`. If that seems applicable to your situation, try reducing the size of the data returned by `getStaticProps()` or move the data fetching step to a later point.

**Reply by lmachens** [reply]
Thx, looks like it was an issue on your side. It was fixed by the engineering team an hour ago.

But I will take a look at the large request size too 👍

**Reply by amyegan** [reply]
It was definitely a problem on our side and different from the original discussion problem. Glad to know it's working now! 🙌
